rest/service/dashboardTemplate.go

```diff
@@ -140,8 +133,8 @@ var (
 		models.NotificationsEvents: models.ModuleFederationMetadata{
 			Scope:       "notifications",
 			Module:      "./DashboardWidget",
+			FeatureFlag: "widget.notifications.enabled",
```

Comment:
@SteveHNH noticed this flag is different than the rest - note `enabled` vs `enable` postfix. Is that intentional?

Reply:
This was an oversight. Good eye.